Updated the archetypes so that the models are correctly accessed from the controller

zhouyaoji opened the pull request.

Commit:
Updated the archetypes so that the models are correctly accessed from the controller.

drewfish commented on lib/app/archetypes/mojit/full/controller.server.js.hb:
```diff
@@ -34,7 +34,5 @@ YUI.add('{{name}}', function(Y, NAME) {
     };
-
-}, '0.0.1', {requires: ['mojito', 'mojito-assets-addon', 'mojito-models-addon', '{{name}}ModelFoo']});
+}, '0.0.1', {requires: ['mojito', 'mojito-assets-addon', 'mojito-models-addon']});
```
Have you tried this?  I think we might still need the `requires` to have `{{name}}ModelFoo`.

zhouyaoji replied:
Yes, I did. I talked to Isao, and he said that `mojito create` uses the archetypes in the `mojito-cli-create` package, so the archetypes in the `mojito` source are obsolete. I'm closing this PR and opening a new one that has removed the `archetypes` directory to avoid future confusion.

Commit:
Removed archetypes from Usage: mojito <command> [options]
The mojito-cli package provides command line helpers for Mojito developers.
Available commands: build, create, doc, help, info, jslint, test, version: they are in  now.
